dist.ini

```diff
@@ -15,13 +15,17 @@ match = ^usr/share/octopussy/CSS/smoothness/images/
 match = ^usr/share/octopussy/HELP/IMG/
 match = ^usr/share/octopussy/IMG/
 encoding = bytes
- 
+
 [GatherDir]
 
 ; Adds META.YML file
 [MetaYAML]
+[MetaJSON]
+[MetaTests]
```

Comment:
Test the META files - which is perhaps slightly silly as they are being auto generated.